### ISSUE TITLE
Known Issue ; builds don't work

### ISSUE THREAD

**Issue by cdhanna** [cdhanna/MarkdownSupportDocs#1]
If you attempt to make a build with _Markdown Support_, you'll see console errors like this,
```
Assets/Markdown/Editor/MarkdownImporter.cs(2,19): error CS0234: The type or namespace name 'AssetImporters' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
```

I am working on a fix.

**Comment by cdhanna** [comment]
Quick update- I have fixed the issue, and submitted a new build to the Unity Asset Store for approval. I expect this fix to be live in a few days.

**Comment by cdhanna** [comment]
this is fixed and released in 1.1
